verboseTree-maven-plugin/src/test/java/org/apache/maven/dependency/graph/DependencyGraphBuilderTest.java

```diff
@@ -17,14 +17,23 @@
 
 package org.apache.maven.dependency.graph;
 
-import org.apache.maven.artifact.Artifact;
-import org.apache.maven.artifact.DefaultArtifact;
+import org.apache.maven.artifact.handler.ArtifactHandler;
+import org.apache.maven.artifact.handler.DefaultArtifactHandler;
 import org.apache.maven.artifact.handler.manager.ArtifactHandlerManager;
 import org.apache.maven.execution.MavenSession;
 import org.apache.maven.plugin.testing.AbstractMojoTestCase;
 import org.apache.maven.project.MavenProject;
 import org.eclipse.aether.DefaultRepositorySystemSession;
+// doesn't work with maven project import org.eclipse.aether.artifact.Artifact;
```

Comment:
try not to include commented code in PRs. It all needs to be removed or uncommented before this can be merged.